src/data/dropdownDataTreeDescriptions.tsx

```diff
@@ -0,0 +1,125 @@
+import {TreeViewData} from '~/components/TreeView/TreeView.types';
+import {Folder, Lock} from '~/icons';
+
+export const dropdownDataTreeDescriptions: TreeViewData[] = [
```

Comment:
It seems we can't display a description with a tree. In that case, we don't need this data and the associate story